broker/src/main/java/io/zeebe/broker/clustering/topology/TopologyManagerImpl.java

```diff
@@ -189,9 +190,17 @@ private BrokerInfo readBrokerInfo(final Member eventSource) {
     final BrokerInfo brokerInfo = BrokerInfo.fromProperties(eventSource.properties());
     if (brokerInfo != null && !isStaticConfigValid(brokerInfo)) {
       LOG.error(
-          "Static configuration of node {} differs from local node {}",
-          eventSource.id(),
-          atomix.getMembershipService().getLocalMember().id());
+          "Static configuration of node {} differs from local node {}." 
```

Comment:
I ran a little manual test, by changing the above if statement to always evaluate to true, locally building the project and running `./dist/target/zeebe-broker/bin/broker`. It outputted the following log statement:
```
2020-10-12 10:44:49.002 [] [atomix-cluster-events] ERROR io.zeebe.broker.clustering - Static configuration of node 0 differs from local node 0.NodeId: 0 <= 0 < 1, ClusterSize: 1 == 1, PartitionsCount: 1 == 1, ReplicationFactor: 1 == 1.
```

So here we see that we also compare our own static configuration to our own configuration 😆. The statement itself is nicely readable and it's clear what would be wrong in a failure case. The only thing we missed was an additional space and it might make sense to change the dot into a colon.

```suggestion
          "Static configuration of node {} differs from local node {}: " 
```